server/network.js

```diff
@@ -420,3 +454,51 @@ network.listNetwork_ = function(networkPath) {
   });
   return ret;
 };
+
+/**
+ * Find all edges connecting the gene to other exist genes.
```

Comment:
A bit more on the naming. The names addGenes and deleteGenes seem to make more sense on the frontend than on the server. The server is stateless and does not remember the previous network. I think "incrementalEdges" or similar names would be much clearer.